packages/api/tests/user.test.js

```diff
@@ -367,8 +367,8 @@ describe('User Tests', () => {
       });
 
       test('Should post then get a valid user and user spotlight', async (done) => {
-        const fakeUser = mocks.fakeUser();
-
+        jest.setTimeout(10000);
+        const fakeUser = await mocks.fakeUser();
```

Comment:
@iperdomo @MwayaB

Tested this without this both with and without the change and it seems to work for me. Generally, I think if we have to change another test something fishy is going on. I am wondering if it is this `await` that doesn't need to be there that necessitates the timeout? 

<img width="695" alt="Screenshot 2023-06-28 at 1 47 59 PM" src="https://github.com/LiteFarmOrg/LiteFarm/assets/30075973/6f4d14da-5fba-4d46-a9e7-585bfe513edf">

Reply:
@Duncan-Brain now that I can run test tests 😅 , same here. I can run the tests without the call to `jest.setTimeout()`